docs/how-to-deploy-scalardb-graphql.md

```diff
@@ -1,8 +1,16 @@
-# How to deploy ScalarDB GraphQL
+# [Deprecated] How to deploy ScalarDB GraphQL
+
+{% capture notice--info %}
+**Note**
+
+ScalarDB GraphQL Server is now deprecated. Please use [ScalarDB Cluster](./how-to-deploy-scalardb-cluster.md) instead of it.
```

Comment:
Just a minor suggestion here. (Changes made here are consistent across other docs in this PR.)

Also, I often see `ScalarDB GraphQL Server` written as `ScalarDB GraphQL server` (with `server` lowercase) or as simply `ScalarDB GraphQL`. Could you please confirm which wording is correct?

```suggestion
ScalarDB GraphQL Server is now deprecated. Please use [ScalarDB Cluster](./how-to-deploy-scalardb-cluster.md) instead.
```

Reply:
Sorry, actually, I don't know the formal name. As I mentioned in our private Slack channel, I'm asking the development team of ScalarDB.

Reply:
According to the discussion, we should use:

* `ScalarDB GraphQL Server` if we mean it as a `server` component of `ScalarDB GraphQL`.
  * e.g., If we mention `node`, we should use `ScalarDB GraphQL Server node` with `Server` capitalized.
* `ScalarDB GraphQL` if we mean it as an `interface` of ScalarDB.